### PR TITLE
Add OneToOneFieldChecker

### PR DESCRIPTION
**Pull request by jorgii**
Solves #735

### CODE REVIEW

File: kiwi_lint/one_to_one_field.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from pylint import checkers
+from pylint.checkers import utils
+
+
+class OneToOneFieldChecker(checkers.BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker,)
+
+    name = 'one-to-one-field-checker'
+
+    msgs = {'E4493': ("Do not use OneToOneField",
```
Comment (atodorov) [inline kiwi_lint/one_to_one_field.py]: Use `R4531` for error code and I will merge after #949 is merged.

**Comment by atodorov** [comment]
@jorgii can you rebase to latest master and resolve the conflicts and apply the minor changes requested so we can merge? Thanks.

File: kiwi_lint/one_to_one_field.py (hunk 1 above)
Comment (jorgii) [reply kiwi_lint/one_to_one_field.py]: Fixed in 8a1c1cc

**Comment by jorgii** [comment]
@atodorov done.

**Comment by atodorov** [comment]
@jorgii for some reason GH doesn't let me merge this. Try `git rebase` instead of `git merge` the latest master branch.

**Comment by jorgii** [comment]
Rebase done

**Comment by atodorov** [comment]
> Rebase done

Definitely not done. Try this one:
https://www.atlassian.com/git/tutorials/merging-vs-rebasing

**Comment by jorgii** [comment]
```
➜  Kiwi git:(features/add-one-to-one-field-checker) ✗ git rebase upstream/master
First, rewinding head to replay your work on top of it...
Applying: Add OneToOneFieldChecker
Using index info to reconstruct a base tree...
M	kiwi_lint/__init__.py
Falling back to patching base and 3-way merge...
Auto-merging kiwi_lint/__init__.py
CONFLICT (content): Merge conflict in kiwi_lint/__init__.py
error: Failed to merge in the changes.
Patch failed at 0001 Add OneToOneFieldChecker
hint: Use 'git am --show-current-patch' to see the failed patch
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
➜  Kiwi git:(52896943) ✗ git rebase --continue
Applying: Add OneToOneFieldChecker
Applying: Use R4531 code for one-to-one-field-checker.
```

That's how I did the rebase. Not really sure what went wrong.

**Comment by atodorov** [comment]
> That's how I did the rebase. Not really sure what went wrong.

Looks like you did not resolve the conflicts. At least I can't tell that you did so.

Try checking out your changes into another branch so you don't loose them. Then go back to the current feature branch, rebase to upstream/master and probably drop the commits which are not relevant/are duplicate.

**Comment by atodorov** [comment]
Cherry-picked, rebased and squashed as #962.

**Pull request closed without merging**